ui/src/components/Models.tsx

```diff
@@ -0,0 +1,60 @@
+export enum RequestStatus{
+    NO_REQUEST="NO REQUEST",
+    IN_PROGRESS = "IN PROGRESS",
+    DENIED = " DENIED",
+    APPROVED = "APPROVED"
+}
+
+export class Teacher{
```

Comment:
this could be done using an interface or type

interface Teacher{
  id: number,
  interest: string,
  enrolledStudents: Student[]
  //tralala
}

//you can use it like so: 
const teacher = {
  id: 5,
  interest: "idk",
  enrolledStudents: [],
} as Teacher